hack/test/pre-upgrade-setup.sh

```diff
@@ -102,6 +102,16 @@ rules:
     - "watch"
     - "bind"
     - "escalate"
+  - apiGroups:
+    - networking.k8s.io
+    resources:
+    - networkpolicies
+    verbs:
+    - get
+    - list
+    - create
+    - update
+    - delete
```

Comment:
Sorry, now I see that we have the NP in both versions
Can we have it in one version and another version, so we ensure that it works with both cases?

Reply:
Did you not mean to approve it? It already merged from your review. Also, not having the NP in one version isn't going to be any different, we always want to be able to support installing NPs so we always want the e2e to try to install one so the CI fails if it can't. There's no value-add from running the same test without an NP.

Reply:
It was a nit idea, I think we may could cover both scenarios 
But it is all fine. I think it is OK as it is no worry.

Reply:
Btw, thank you a lot for taking care of that !!!